client/layout/guided-tours/main-tour.js

```diff
@@ -200,11 +190,11 @@ export const MainTour = makeTour(
 					} )
 				}
 			</p>
-			<div className="guided-tours__single-button-row">
+			<ButtonRow single={ true }>
```

Comment:
Aha, so this is what `single` is for. Hm, the component should be able to know this. Off the top of my mind, something like

```js
( { single, children } ) => {
  const isSingleButton = React.Children.count( children ) === 1;
  return (
    <div …
```